DEVELOPMENT.md

```diff
@@ -0,0 +1,84 @@
+# Development
+
+This doc explains how to set up a development environment, so you can get started
+contributing to `oam-kubernetes-runtime` or build a PoC (Proof of Concept). 
+
+## Prerequisites
+
+1. Golang version 1.12+
+2. Kubernetes version v1.15+ with kubectl configured
+
+## Build
+
+The functionality of this library can be demonstrated with the following steps:
+
+* Fork and clone this project
+```shell script
+git clone https://github.com/xxxxxx/oam-kubernetes-runtime.git
+```
+
+* Build the library 
+
+```shell
+make submodules 
+
+make
```

Comment:
Maybe we can copy-paste the output together. Developers will be more confirmed if they have the same output.

Reply:
+1